pages/api/getApplications.ts

```diff
@@ -0,0 +1,25 @@
+import { getApplicationsFromDb } from "@/lib/database";
+import { getServerSession } from "next-auth/next";
+import { authOptions } from "./auth/[...nextauth]";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<any>,
+) {
+  const session = await getServerSession(req, res, authOptions);
+
+  if (session) {
```

Comment:
This looks like it allows any user with a login session to load all applications. That is a privacy problem. Are these all types of applications (company and employee)? Can we have some sort of permission system here? Registrars see all company applications. Companies can see only their employee applications.

Reply:
Don't need this file. Removing.